src/csharp/Microsoft.Spark/Utils/JvmObjectUtils.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+
+namespace Microsoft.Spark.Utils
+{
+    /// <summary>
+    /// JvmObjectUtils provides functions related to JVM objects.
+    /// </summary>
+    internal static class JvmObjectUtils
```

Comment:
I think this name could be too generic. What do you want to achieve thru this class in a long run? Will this be more like a factory?

Reply:
How about putting  `jvm.CallConstructor("java.util.ArrayList");` and `jvm.CallConstructor("java.util.Hashtable");` here as well?